Makefile

```diff
@@ -23,59 +23,59 @@ test-short:
 
 # run E2E tests
 test-e2e:
-	go run ./tests/e2e/...
+	go run ./e2e/...
 
 # run difference tests
 test-diff:
 	go test ./tests/difference/...
 
 # run only happy path E2E tests
 test-e2e-short:
-	go run ./tests/e2e/... --tc happy-path
+	go run ./e2e/... --tc happy-path
 
 # run only happy path E2E tests with cometmock
 # this set of traces does not test equivocation but it does check downtime
 test-e2e-short-cometmock:
-	go run ./tests/e2e/... --tc happy-path-short --use-cometmock --use-gorelayer
+	go run ./e2e/... --tc happy-path-short --use-cometmock --use-gorelayer
 
 # run full E2E tests in sequence (including multiconsumer)
 test-e2e-multi-consumer:
-	go run ./tests/e2e/... --include-multi-consumer
+	go run ./e2e/... --include-multi-consumer
 
 # run full E2E tests in parallel (including multiconsumer)
 test-e2e-parallel:
-	go run ./tests/e2e/... --include-multi-consumer --parallel
+	go run ./e2e/... --include-multi-consumer --parallel
 
 # run full E2E tests in sequence (including multiconsumer) using latest tagged gaia
 test-gaia-e2e:
-	go run ./tests/e2e/... --include-multi-consumer --use-gaia
+	go run ./e2e/... --include-multi-consumer --use-gaia
 
 # run only happy path E2E tests using latest tagged gaia
 test-gaia-e2e-short:
-	go run ./tests/e2e/... --tc happy-path --use-gaia
+	go run ./e2e/... --happy-path-only --use-gaia
 
 # run full E2E tests in parallel (including multiconsumer) using latest tagged gaia
 test-gaia-e2e-parallel:
-	go run ./tests/e2e/... --include-multi-consumer --parallel --use-gaia
+	go run ./e2e/... --include-multi-consumer --parallel --use-gaia
 
 # run full E2E tests in sequence (including multiconsumer) using specific tagged version of gaia
 # usage: GAIA_TAG=v9.0.0 make test-gaia-e2e-tagged
 test-gaia-e2e-tagged:
-	go run ./tests/e2e/... --include-multi-consumer --use-gaia --gaia-tag $(GAIA_TAG)
+	go run ./e2e/... --include-multi-consumer --use-gaia --gaia-tag $(GAIA_TAG)
 
 # run only happy path E2E tests using latest tagged gaia
 # usage: GAIA_TAG=v9.0.0 make test-gaia-e2e-short-tagged
 test-gaia-e2e-short-tagged:
-	go run ./tests/e2e/... --tc happy-path --use-gaia --gaia-tag $(GAIA_TAG)
+	go run ./e2e/... --happy-path-only --use-gaia --gaia-tag $(GAIA_TAG)
```

Comment:
same here, what is the reason to go back to the 'old' flag and not using --tc
if I'm not mistaken that's not even supported anymore. did you test that?

Reply:
That test has not been working for a while. Gaia is on v45, the ICS main is on v47.
Since e2e was coupled to the ICS version, some of the commands to execute against a gaia node are not valid and cannot be executed. Notice that most of that was intended for `gaia v9`.

Reply:
Changed, thanks!